apps/codelab/src/app/codelabs/angular/create-first-app/output.json

```diff
@@ -0,0 +1,43 @@
+{
+  "s":[
+    {
+      "correct": 
+      [
+        {"htmlStr": "<div class=\"x\"></div>"},
+        {"htmlStr": "<div class=\"y\"></div>"},
+        {"htmlStr": "<div class=\"c\"></div>"}
+      ],
+      "incorrect": 
+      [
+        {"htmlStr": "<div></div>"},
+        {"htmlStr": "<span></span>"},
+        {"htmlStr": "<p></p>"}
+      ]
+    },
+    {
+      "correct": 
+      [
+        {"htmlStr": "<div class=\"q1\"></div>"},
```

Comment:
Can you keep it as a sting as opposed to an object?

Reply:
No, I add properties to the object to keep track of whether or not to highlight that line

Reply:
Don't do that